evalai/utils/common.py

```diff
@@ -0,0 +1,34 @@
+from click import echo
+
+
+class Colors:
```

Comment:
Why not use `crayons`, or better yet, `click` supports `colorama`. See [this](http://click.pocoo.org/5/utils/#ansi-colors).
Different shells tend to not like this sort of hard definition of color strings.

Reply:
I didn't use external libraries because it didn't support bold font within the terminal. So I thought to define it within utils. Should I add a library just for the colours and then use bold font defined separately?

Reply:
`click`'s style method supports bold font. You should check it out.
In that case, using `colorama` seems be to your best bet.

Reply:
OMG! It works perfectly! Thanks for the suggestion! 😄

Reply:
Makes life a whole lot easier, right? :slightly_smiling_face:

Reply:
Definitely! 👍